rasterio/rpc.py

```diff
@@ -93,11 +93,12 @@ def from_gdal(cls, rpcs):
         out = {}
 
         for key, val in rpcs.items():
-            vals = val.split()
-            if len(vals) > 1:
-                out[key] = [float(v) for v in vals]
+            # Four items have 20 floats in their values.
+            if key in {"LINE_NUM_COEFF", "LINE_DEN_COEFF", "SAMP_NUM_COEFF", "SAMP_DEN_COEFF"}:
+                out[key] = [float(v) for v in val.split()[:20]]
```

Comment:
```suggestion
                out[key] = [float(v) for v in val.split(maxsplit=20)[:20]]
```
https://docs.python.org/3/library/stdtypes.html#str.split

Reply:
Thanks! The suggested changes are pushed now.